integration-api-default-config.json

```diff
@@ -727,8 +727,14 @@
 							"expr": "name"
 						},
 						"values": [
+							{
+								"expr": "${lx.factsheet.name}"
+							},
 							{
 								"expr": "${integration.valueOfForEach}"
+							},
+							{
+								"expr": "${integration.valueOfForEach}(Copy ${integration.now.getYear()}-${integration.now.getMonthValue()<10?'0':''}${integration.now.getMonthValue()}-${integration.now.getDayOfMonth()<10?'0':''}${integration.now.getDayOfMonth()} ${integration.now.getHour()<10?'0':''}${integration.now.getHour()}:${integration.now.getMinute()<10?'0':''}${integration.now.getMinute()})"
 							}
```

Comment:
just for my understanding.! Where can we find this expression template api?. Is there documentation for this?
For Eg: integration.now.getMinute()

Reply:
@SudharmaPuranikLeanIX https://docs.leanix.net/docs/advanced#using-integration-object

Reply:
Thank you :) @daniellesarmiento